server/metrics/metrics.go

```diff
@@ -166,6 +168,20 @@ func New(config *libconfig.MetricsConfig) *Metrics {
 			},
 			[]string{"name"},
 		),
+		metricProcessesDefunctTotal: prometheus.NewGaugeFunc(
+			prometheus.GaugeOpts{
+				Subsystem: collectors.Subsystem,
+				Name:      collectors.ProcessesDefunctTotal.String(),
+				Help:      "Total number of defunct processes in the node",
+			},
+			func() float64 {
+				total, err := process.DefunctProcesses()
+				if err == nil {
+					return float64(total)
+				}
```

Comment:
it doesn't look like we logged the error here

Reply:
@haircommander Should a `logrus.Warnf("%v", err)` before `return 0` in line 182 suffice?

Reply:
yup that would be good!